src/constants.ts

```diff
@@ -1,25 +1,48 @@
 import JSBI from 'jsbi'
 
-export enum SupportedChainId {
+export enum ChainId {
   MAINNET = 1,
+  ROPSTEN = 3,
+  RINKEBY = 4,
   GOERLI = 5,
   SEPOLIA = 11155111,
-
-  ARBITRUM_ONE = 42161,
-  ARBITRUM_GOERLI = 421613,
-
+  KOVAN = 42,
   OPTIMISM = 10,
   OPTIMISM_GOERLI = 420,
-
+  OPTIMISTIC_KOVAN = 69,
+  ARBITRUM_ONE = 42161,
+  ARBITRUM_RINKEBY = 421611,
+  ARBITRUM_GOERLI = 421613,
   POLYGON = 137,
   POLYGON_MUMBAI = 80001,
-
   CELO = 42220,
   CELO_ALFAJORES = 44787,
-
+  GNOSIS = 100,
+  MOONBEAM = 1284,
   BNB = 56
 }
 
+export const SUPPORTED_CHAINS = [
+  ChainId.MAINNET,
+  ChainId.RINKEBY,
```

Comment:
you can get rid of some of these chains - theyre gone forever

Reply:
rinkeby, ropsten (i think on L2s too), and kovan as well - all test networks that dont exist anymore

Reply:
@lynnshaoyu I brought these chains over from the BE repo, can we deprecate across the org?

Reply:
confirmed offline that these can be removed